wordpress-mobile/WordPress-Android
# [JP Install Full Plugin] Enable feature flag and update release notes

Fixes #17837 

To test:
- Install app and select site with only individual plugins;
- Open home screen: the JP full plugin install card should be shown.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
Feature config files are currently not testable.

PR submission checklist:
